core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.controller.spec.ts

```diff
@@ -124,16 +124,15 @@ describe('Edit Thumbnail Modal Controller', function() {
     // The setTimeout is being used here to not conflict with $timeout.flush
     // for fadeIn Jquery method. This first setTimeout is to wait the default
     // time for fadeOut Jquery method to complete, which is 400 miliseconds.
-    // 800ms is being used instead of 400ms just to be sure that fadeOut callbak
-    // is already executed.
+    // 1000ms is being used instead of 400ms just to be sure that fadeOut
+    // callback is already executed.
```

Comment:
Should we increase more than 1000 if this is causing issues? How did you test that it is not flaky anymore? I would suggest replicating this test a lot of times (in a loop or something) and running that.

Reply:
Sure, I'll do that.

Reply:
Tested this 1000x times
![Screenshot from 2021-04-21 12-23-52](https://user-images.githubusercontent.com/54100702/115509790-7e26c500-a29c-11eb-904d-bd991bc450ec.png)

Reply:
The error happens when all the tests are run together in most cases though. Can you try that?